src/probnum/filtsmooth/gaussfiltsmooth/_utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Utility functions for Gaussian filtering and smoothing.
+"""
+
+from probnum.filtsmooth.statespace import (
+    ContinuousModel,
+    DiscreteModel,
+    LinearSDEModel,
+    DiscreteGaussianLinearModel,
+)
+
+
+def is_cont_disc(dynamod, measmod):
+    """Checks whether the state space model is continuous-discrete."""
+    dyna_is_cont = issubclass(type(dynamod), ContinuousModel)
```

Comment:
`isinstance(dynamod, ContinuousModel)` should also work and might be more concise.

Reply:
Ad hoc, I dont think it would work. LinearSDE models for instance are subclasses of ContinuousModel.

Reply:
I'm not sure I understand your point. Do you have an example of something that `issubclass(type(variable), Class)` does that `isinstance(variable, Class)` does not? To my understanding both are equivalent, but I am definitely not sure about this :sweat_smile:

Reply:
I was convinced that it is different, checked, and it turns out you are right.